src/DynamoCoreWpf/Services/LoginService.cs

```diff
@@ -56,7 +53,7 @@ public bool ShowLogin(object o)
             return navigateSuccess;
         }
 
-        internal void ShowShapewaysLogin(ShapewaysClient client) 
+        internal void ShowShapewaysLogin(ShapewaysClient.ShapewaysClient client) 
```

Comment:
I am ok with this. But it looks better if there is an alias created for shapeways namespace.